pauljoegeorge/m5stickc-mrubyc-template
# [WIP] BLE GATT server

# Steps
  - Install nRF Connect Mobile App
  - Git pull the latest code
  - Run `make menuconfig`
    - `enable` Bluetooth
 - Run make flash monitor
 - Open nRF Connect Mobile App
 - Look for Bluetooth host named `ESP_DOOR_CHIME`
 - Connect to `ESP_DOOR_CHIME`
 - Swipe Right and click  and you will see available characteristics
 - Click on write button, write BOOLEAN SIGNAL
 - if BOOLEAN is set to TRUE, confirm that M5StickC display change to BLUE
